Apps/Sandcastle/gallery/Materials.html

```diff
@@ -218,17 +218,25 @@
           });
         }
 
-        function applyCompressedTextureMaterial(primitive, scene) {
-          Sandcastle.declare(applyCompressedTextureMaterial); // For highlighting in Sandcastle.
-
-          var compressedImageUrl;
-          if (scene.getCompressedTextureFormatSupported("s3tc")) {
-            compressedImageUrl = "../images/LogoDXT1.ktx";
-          } else if (scene.getCompressedTextureFormatSupported("etc1")) {
-            compressedImageUrl = "../images/LogoETC1.ktx";
-          } else if (scene.getCompressedTextureFormatSupported("pvrtc")) {
-            compressedImageUrl = "../images/LogoPVR.ktx";
-          }
+        function applyETC1SCompressedTextureMaterial(primitive, scene) {
+          Sandcastle.declare(applyETC1SCompressedTextureMaterial); // For highlighting in Sandcastle.
+
+          var compressedImageUrl = "../images/Cesium_Logo_ETC1S.ktx2";
+
+          primitive.appearance.material = new Cesium.Material({
+            fabric: {
+              type: "Image",
+              uniforms: {
+                image: compressedImageUrl,
+              },
+            },
+          });
+        }
+
+        function applyUASTCCompressedTextureMaterial(primitive, scene) {
+          Sandcastle.declare(applyUASTCCompressedTextureMaterial); // For highlighting in Sandcastle.
+
+          var compressedImageUrl = "../images/Cesium_Logo_UASTC.ktx2";
```

Comment:
`Apps/Sandcastle/images/Cesium_Logo_Flat.ktx2` can be deleted as it is no longer being used in the materials sandcastle